src/pages/[platform]/build-a-backend/data/set-up-data/index.mdx

```diff
@@ -81,10 +78,30 @@ The `a.allow.public()` rule designates that anyone authenticated using an API ke
 
 To deploy these resources to your cloud sandbox, run the following CLI command in your terminal:
 
+<InlineFilter filters={["react", "angular", "javascript", "vue", "nextjs", "react-native"]}>
+
 ```bash
 npx amplify sandbox
 ```
 
+</InlineFilter>
+
+<InlineFilter filters={["android"]}>
+
+```bash
```

Comment:
I don't think `--config-format json-mobile` is needed. May not even be an option for GA.